radar-upload-backend/src/main/java/org/radarbase/upload/inject/DoaEntityManagerFactory.kt

```diff
@@ -13,11 +13,11 @@ import javax.persistence.EntityManager
 import javax.persistence.Persistence
 import javax.ws.rs.core.Context
 
-class DoaEntityManagerFactory(@Context config: Config, jpaConfig: Map<String, String> = emptyMap()) : DisposableSupplier<EntityManager> {
+class DoaEntityManagerFactory(@Context config: Config) : DisposableSupplier<EntityManager> {
```

Comment:
Why remove the jpaConfig variable? This can be useful in testing and profiling.

Reply:
The initialization does not work with emptyMap(). I propose we add those as additional settings in config and load them, it needed

Reply:
In that case, you can also use the Kotlin `class DoaEntityManagerFactory @JvmOverloads constructor(...)` annotation to ensure that it has separate entities. But a Config map property works as well.